validator/src/main/java/com/google/daq/mqtt/util/ConfigDiffEngine.java

```diff
@@ -30,34 +30,44 @@ public ConfigDiffEngine() {
    */
   @SuppressWarnings("unchecked")
   public List<String> computeChanges(Config deviceConfig) {
-    Map<String, Object> updated = convertSemantics(deviceConfig);
+    Map<String, Object> updated = extractDefinitions(deviceConfig);
```

Comment:
What's our current stance with "unit testing" coverage?  I'd say that if we want to improve increasing our unit testing in general that now would be the time to do it (i.e. at least a >0 incremental improvement in test overage)... ?

Reply:
We should have tests as much as possible for new or changed code. AFAIK this change comes from a different change. Do you want to add tests in a different PR?